staging/src/k8s.io/apiserver/plugin/pkg/authenticator/token/webhook/webhook.go

```diff
@@ -171,7 +171,11 @@ type tokenReviewClient struct {
 }
 
 func (t *tokenReviewClient) Create(tokenReview *authentication.TokenReview) (*authentication.TokenReview, error) {
```

Comment:
why is a context not passed from callsite as we've talked about doing in a future client-go?  I accept that this is congruent and would allow compatibility.  You would choose that over simply say, "new version, if you want it, update"?

Reply:
This is to ease update from one version of the library to the next